idunn/api/instant_answer.py

```diff
@@ -223,34 +229,62 @@ async def get_instant_answer(
     if lang in nlu_allowed_languages:
         try:
             intentions = await nlu_client.get_intentions(
-                normalized_query, lang, extra_geocoder_params
+                normalized_query,
+                lang,
+                extra_geocoder_params,
+                allow_types=[
+                    IntentionType.BRAND,
+                    IntentionType.CATEGORY,
+                    IntentionType.POI,
+                    IntentionType.ANY_PLACE,
```

Comment:
As this intention type `ANY_PLACE` is very common and could have a significant impact on response times, I wonder if we could remove it from this list for now, and hopefully improve the coverage of intentions with future iterations of the tagger?

@remi-dupre  What do you think ?

Reply:
I'll change it, especially since this a rather niche improvement :/